test/hotspot/jtreg/runtime/SharedArchiveFile/MaxMetaspaceSize.java

```diff
@@ -24,11 +24,13 @@
 /**
  * @test
  * @requires vm.cds
+ * @requires vm.flagless
  * @bug 8067187 8200078
  * @summary Testing CDS dumping with the -XX:MaxMetaspaceSize=<size> option
  * @library /test/lib
  * @modules java.base/jdk.internal.misc
  *          java.management
+ * @run driver MaxMetaspaceSize
  */
```

Comment:
Includes the changes from [8243945](https://bugs.openjdk.org/browse/JDK-8243945) on this file